packages/boot/src/__tests__/acceptance/application-metadata.booter.acceptance.ts

```diff
@@ -33,14 +29,18 @@ describe('application metadata booter acceptance tests', () => {
     // Add the following files
     // - package.json
     // - dist/application.js
+
+    const appJsFile = resolve(__dirname, '../fixtures/application.js');
+    let appJs = fs.readFileSync(appJsFile, 'utf-8');
+    // Adjust the relative path for `import`
+    appJs = appJs.replace('../..', '../../..');
```

Comment:
IIUC, we are trying to fix the following line:

https://github.com/strongloop/loopback-next/blob/4e5f721ca666ab5047f4c36fbcb36a39b76966bb/packages/boot/src/__tests__/fixtures/application.ts#L10

An idea to consider: instead of changing the import/require path in the application file copied to sandbox, can we put a short `index.js` file into the directory where `../..` points to, to make `require('../..')` work again?

```js
// index.js
module.exports = require('../');
```

Reply:
I prefer to use `transform` to make it clear - having a magic `index.js` makes it difficult to understand.